README.md

```diff
@@ -316,7 +308,7 @@ You can find details on the presets DevTools supports for Slow and Fast 3G in th
 
 <h3 id="throttle-network-cpu">Emulate a slow network and CPU</h3>
 
-CPU throttling allows you to simulate how a page performs on slower mobile devices. This can be done using `Network.emulateNetworkConditions` via the Chrome DevTools Protocol.
+CPU throttling allows you to simulate how a page performs on slower mobile devices. This can be done using `page.emulateNetworkConditions` via the Chrome DevTools Protocol.
```

Comment:
```suggestion
CPU throttling allows you to simulate how a page performs on slower mobile devices. This can be done using Puppeteer’s `page.emulateNetworkConditions` API.
```